src/main/java/com/google/codeu/servlets/UserLevelServlet.java

```diff
@@ -46,28 +46,32 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         response.getOutputStream().println(json);
     }
 
-    /** Gets called when a user presses the 'Level Up' button - increments the users level
+    /** Gets called when a user presses the 'Submit' button - increments the users level
      * then reloads the page! */
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
         UserService userService = UserServiceFactory.getUserService();
         String user_email = userService.getCurrentUser().getEmail();
         User user = datastore.getUser(user_email);
         int level = user.getLevel();
-        String answer;
+        String answer = "";
```

Comment:
We could probably change the answer to upper case up here instead of calling it twice inside the if/else statements

Reply:
I'm not sure we can do it on the same line because the input is different and is processed in uppercase form.